src/java/org/apache/cassandra/cql3/statements/schema/AlterKeyspaceStatement.java

```diff
@@ -99,6 +102,15 @@ public void authorize(ClientState client)
         client.ensureKeyspacePermission(keyspaceName, Permission.ALTER);
     }
 
+    @Override
+    public void validate(ClientState state)
```

Comment:
can you remove this method?  its defined to do this in `org.apache.cassandra.cql3.statements.schema.AlterSchemaStatement#validate`